code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3289,6 +3289,45 @@ datum
 			thirst_value = 1
 			bladder_value = -1
 
+		fooddrink/turmeric
+			name = "turmeric powder"
+			id = "turmpowder"
+			description = "Has a warm, spicy scent and nausea-soothing properties. But if you get this shit on something, the stain's NEVER coming out."
+			reagent_state = SOLID
+			fluid_r = 224
+			fluid_g = 168
+			fluid_b = 12
+			transparency = 255
+
+			reaction_mob(var/mob/M, var/method = TOUCH, var/volume)
+				. = ..()
+				if(method == TOUCH)
```

Comment:
I would require a minimum volume for these reactions, something that would mean that smoking this stuff isn't likely to turn the whole shuttle a funny color.